tests/playwright/src/model/pages/build-image-page.ts

```diff
@@ -43,9 +46,16 @@ export class BuildImagePage extends BasePage {
     this.containerFilePathButton = page.getByRole('button', { name: 'Browse...' }).first();
     this.arm64Button = page.getByLabel('linux/arm64');
     this.amd64Button = page.getByLabel('linux/amd64');
+    this.arm64checkbox = page.getByLabel('ARM® aarch64 systems');
```

Comment:
Can you please also add another locator that would hold the div/container where all these buttons / checkboxes are? 
there is this container: 
```
<div class="flex flex-col" role="region" aria-label="Build Platform Options">
```
that contains all the archs that could be selected.

another question, why it is that we need to add checkbox when there are already existing buttons (`arm64button`) that if you click on them, the checkbox is being toggled?

Reply:
I'm not sure I understand, why do we need to grab the parent element here? What is the point of that since we can't do anything with it anyway?

As for your second question... yes the checkboxes are being toggle when the buttons are being clicked, so even though to change toggle states you need to click the buttons, to check the current state of the checkbox you need to validate the checked/unchecked status of the checkbox, not the button.

Reply:
> I'm not sure I understand, why do we need to grab the parent element here? What is the point of that since we can't do anything with it anyway?

To avoid ambiguity. In future the page can be changed so that we make sure to search for the element in the right place. Also, reading: `page.getByLabel()` in the code is pretty vague. You have to search through whole page. Whilst with ie. `platform.getByLabel` where `platform` is the locator that represents a div/container or region in this case to hold all the elements tied to a platform selection or adding new one, etc. So it is true that you cannot do anything with it (except checking ie. visibility), but it helps navigation. 

> As for your second question... yes the checkboxes are being toggled when the buttons are being clicked, so even though to change toggle states you need to click the buttons, to check the current state of the checkbox you need to validate the checked/unchecked status of the checkbox, not the button.

True! good point.

Reply:
Ok, I extracted the platform region in a field and using it when getting the platform buttons, but to be honest to me it doesn't make much sense given that each button/checkbox is specifically named, I don't really think it helps with debugging at all, to me it seems the same only that we're using an intermediate element instead of going to the element needed.